build.sh

```diff
@@ -2,5 +2,5 @@
 sed -i '/^### DATA ###$/q' wofi-emoji
 
 curl https://raw.githubusercontent.com/muan/emojilib/main/dist/emoji-en-US.json \
```

Comment:
I think we should pin the last release here for reproducible builds and prevent unexpected breakages. WDYT?